File: module-web/src/main/java/maple/expectation/controller/dto/donation/SendCoffeeResponse.java
package maple.expectation.controller.dto.donation;

/**
 * 커피 후원 응답 DTO
 *
 * @param requestId 멱등성 키 (중복 요청 방지용)
 * @param message 성공 메시지
 */
public record SendCoffeeResponse(String requestId, String message) {
  public static SendCoffeeResponse success(String requestId) {
    return new SendCoffeeResponse(requestId, "커피 후원이 완료되었습니다.");
  }
}

File: module-web/src/main/kotlin/maple/expectation/web/dto/donation/SendCoffeeResponse.kt
package maple.expectation.web.dto.donation

/**
 * 커피 후원 응답 DTO
 *
 * @param requestId 멱등성 키 (중복 요청 방지용)
 * @param message 성공 메시지
 */
data class SendCoffeeResponse(
    val requestId: String,
    val message: String,
) {
    companion object {
        fun success(requestId: String): SendCoffeeResponse = SendCoffeeResponse(requestId, "커피 후원이 완료되었습니다.")
    }
}
